now/data_loading/data_loading.py

```diff
@@ -48,6 +49,21 @@ def load_data(user_input: UserInput, data_class=None) -> DocumentArray:
     return da
 
 
+def get_da_with_index_fields(da: DocumentArray, user_input: UserInput):
+    for d in da:
+        temp_da = []
+        for field in user_input.index_fields:
+            _index_field_doc = getattr(d, field)
+            temp_da.append(_index_field_doc)
+        d.chunks = DocumentArray(temp_da)
```

Comment:
I think this should also work?
```suggestion
        d.chunks = [getattr(d, field) for field in user_input.index_fields]
```